docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -104,12 +104,23 @@ storage intact and b) you can build software before you deploy your cluster.
    make
    ```
 
-1. Generate the deployment folder after replacing `<project>` with the project id.
+1. Generate the deployment folder after replacing `<project>` with the project
+   id.
+
+   If you are running this as a test, and don't care about the files created in
+   the cloud buckets being destroyed, it is recommneded you run:
 
    ```bash
-   ./ghpc create docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml -w --vars project_id=<project>
+   ./ghpc create docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml -w --vars project_id=<project> --vars bucket_force_delete=true
    ```
 
+   The `bucket_force_delete` variable makes it easier to tear down the
+   deployment. If it is set to `false`, buckets with objects (files) will not be
```

Comment:
nit: I would call out that this is the default: 
`If it is set to the default value of `false`, buckets with objects...`